pydantic/functional_validators.py

```diff
@@ -190,10 +190,15 @@ def __get_pydantic_core_schema__(self, source_type: Any, handler: GetCoreSchemaH
 
         try:
             schema = handler(source_type)
-            serialization = core_schema.wrap_serializer_function_ser_schema(
-                function=lambda v, h: h(v),
-                schema=schema,
-                return_schema=schema,
+            # TODO if `schema['serialization']` is one of `'include-exclude-dict/sequence',
+            # schema validation will fail. That's why we use 'type ignore' comments below.
+            serialization = schema.get(
+                'serialization',
+                core_schema.wrap_serializer_function_ser_schema(
+                    function=lambda v, h: h(v),
+                    schema=schema,
+                    return_schema=schema,
```

Comment:
Do we not need to do `handler.generate_schema(source_type)` here?

Reply:
Hum I don't think it will change anything in this case as `schema` is guaranteed to not have any serialization key here, but I'll change to align with how it is done in `Plain/WrapSerializer`.